### DISCUSSION TITLE
Virus on Releases?

### DISCUSSION THREAD

**Discussion by Joaquin500** [community/community#17216] [Projects and Issues]
When I try to upload a binary while creating a new Release, my antivirus closes the connection saying that the server is infected by URL:Blacklist (see attached image). Am I doing something wrong or is the server actually infected?

![GitHub issue](https://user-images.githubusercontent.com/105786359/170022280-de60df4d-2782-4b75-9203-8019c8590bca.png)
?

**Comment by i-marsh** [comment]
Could just be a false positive; AVG blocking the githubusercontent(.)com 
you can follow these instructions for adding an exception to AVG https://support.avg.com/SupportArticleView?l=en&urlName=avg-antivirus-scan-exclusions&q=how+to+make+exclusions+from+all+scans+and+shields&supportType=home
